HISTORY.md

```diff
@@ -1,3 +1,36 @@
+## 11.0.0
```

Comment:
This is not the right way. HISTORY.md is taken and then any newer version is added, so you'll wipe all the history between 11.0.0 and 3.2.0 this way. See https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog for that.

What you're looking for is https://github.com/github-changelog-generator/github-changelog-generator#using-the-summary-section-feature to add some manual notes to a version.